docs/docs/prompts/index.md

```diff
@@ -0,0 +1,145 @@
+# Prompts
+
+Prompts are instructions for Large Language Models (LLMs) that guide them in generating responses.
+They define the content and structure of your interactions with LLMs.
+This section describes how to create and run prompts with Koog.
+
+## Creating prompts
+
+In Koog, all prompts are represented as [**Prompt**](https://api.koog.ai/prompt/prompt-model/ai.koog.prompt.dsl/-prompt/index.html)
+objects. A Prompt object contains:
+
+- **ID**: A unique identifier for the prompt.
+- **Messages**: A list of messages that represent the conversation with the LLM.
+- **Parameters**: Optional [LLM configuration parameters](https://api.koog.ai/prompt/prompt-model/ai.koog.prompt.params/-l-l-m-params/index.html)
+  (such as temperature, tool choice, and others).
+
+All Prompt objects are structured prompts defined using the Kotlin DSL, which lets you specify the structure of the conversation.
+
+!!! note
+    AI agents let you provide a simple text prompt instead of creating a Prompt object.
```

Comment:
I am not sure it's correct to say that AI agents "provide a simple text prompt". Rather, they can take a text string as input